src/backend/distributed/planner/multi_router_planner.c

```diff
@@ -2275,7 +2275,7 @@ CreateDummyPlacement(bool hasLocalRelation)
 		return CreateLocalDummyPlacement();
 	}
 
-	List *workerNodeList = ActiveReadableNonCoordinatorNodeList();
+	List *workerNodeList = ActiveReadableNodeList();
```

Comment:
one line above, we return local dummy placement when needed. Why do we need coordinator here?

Reply:
Right, I guess in terms of functionality including/excluding coordinator doesn't make a difference since this is just a dummy placement, but I will revert this since it seems unnecessary.

Reply:
I think changing this line is an improvement. Otherwise if the coordinator is part of the workers it's never part of round-robin (unless it's the only node), which seems weird.

Reply:
I guess we won't do this improvement in this PR if there isn't any objection.